dump-objects

```diff
@@ -134,7 +139,7 @@ extract_versioned_apis() {
 }
 
 validate_kinds() {
-  local -a expected=( $(< /usr/local/share/k8s-object-dumper/must-exist) )
+  local -a expected=( "$(< /usr/local/share/k8s-object-dumper/must-exist)" )
```

Comment:
I know shellcheck complains about this, but is the behavior the same?

Reply:
Yes tested it locally.

Reply:
Fixed by using mapfile